lib/engine/game/g_1850/step/track.rb

```diff
@@ -69,12 +69,20 @@ def buy_mesabi_token(corporation)
             corporation.mesabi_token = true
           end
 
+          def hex_neighbors(entity, hex)
+            connected = super
+            @game.clear_token_graph_for_entity(entity)
+            connected
```

Comment:
I had to add graph clearing as i ran into an issue:
The hex neighbors check caches the graph when it runs, however it seem to happen prior to a corporation placing its home token when floating, resulting in the corp not being able to lay a tile.
This seems like a bad idea, as it will mean often refreshes of the graph, and bad performance.
Is there a better place to clear the graph? maybe right after home token placement?